### DISCUSSION TITLE
Invitation to Collaborate

### DISCUSSION THREAD

**Discussion by ceford** [max123kl/J4.docs#1] [General]
I received an invitation to collaborate but when I clicked the link I got a message:

` Sorry, we couldn't find that repository invitation. It is possible that the invitation was revoked or that you are not logged into the invited account.  `

So I thought I would try out this discussion board.

**Comment by ceford** [comment]
It is showing the a red background above the list of discussions. Why is that?

**Reply by max123kl** [reply]
send you an email

**Comment by max123kl** [comment]
@carlitorweb did you get an invitation to join here?

**Reply by carlitorweb** [reply]
Yes
